### PR TITLE
TrueReach Bidder Adapter

### PR DESCRIPTION
**Pull request by mmprebid**
## Type of change

- [x] New bidder adapter   

## Description of change

- test parameters for validating bids
```
{
  bidder: 'truereach',
  params: {
     site_id: '0142010a-8400-1b01-72cb-a553b9000009',
     bidfloor: 0.1
  }
}
```

- contact email of the adapter’s maintainer : mm.github@momagic.com
- [x] official adapter submission

Related [prebid.github.io PR Link](https://github.com/prebid/prebid.github.io/pull/2099)

### CODE REVIEW

**Comment by patmmccann** [comment]
Can you copy adomain from your response to bid.meta.advertiserDomains as per http://prebid.org/dev-docs/bidder-adaptor.html#interpreting-the-response

**Comment by mmprebid** [comment]
@patmmccann we have added _**advertiserDomains**_ in the new commit. Please verify.

**Comment by mmprebid** [comment]
@patmmccann / @idettman / @mike-chowla  Kindly let us know the review progress or any issues.

**Comment by bretg** [comment]
@idettman is backed up, so assigning over to @Fawke

File: modules/truereachBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
+import * as utils from '../src/utils.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { config } from '../src/config.js';
+import { BANNER } from '../src/mediaTypes.js';
+
+const SUPPORTED_AD_TYPES = [BANNER];
+const BIDDER_CODE = 'truereach';
+const BIDDER_URL = '//ads.momagic.com/exchange/openrtb25/';
```
Comment (Fawke) [inline modules/truereachBidAdapter.js:8]: BIDDER_URL must use `https`

Comment (mmprebid) [reply modules/truereachBidAdapter.js]: @Fawke We have updated BIDDER_URL to use `https` always. Please verify.

**Pull request merged**